docs/rfcs/0001-server-side-transaction-execution.md

```
@@ -0,0 +1,489 @@
+# Feature enablement
+GUARDIAN_TX_PROVER_URL=https://tx-prover.testnet.miden.io
+GUARDIAN_TX_PROVER_TIMEOUT_SECS=300
+GUARDIAN_PROVING_ENABLED=true
+
+# Execution & Lease Controls
+GUARDIAN_EXECUTION_LEASE_SECS=120
+GUARDIAN_EXECUTION_RECONCILE_INTERVAL_SECS=30
+```
+
+> **Warning for Operators:** The upstream prover client's own default timeout (10 seconds) is below real testnet proving times (6–20 s), which is why the proposed Guardian default is far higher. If you override `GUARDIAN_TX_PROVER_TIMEOUT_SECS`, keep it well above observed proving times for your prover.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)